sources/packages/web/src/views/aest/institution/Profile.vue

```diff
@@ -81,11 +81,11 @@
       <p class="category-header-medium mt-5">Mailing address</p>
       <content-group>
         <title-value
-          propertyTitle="Address 1"
+          propertyTitle="Address line 1"
           :propertyValue="institutionProfileDetail.mailingAddress?.addressLine1"
```

Comment:
Not a part of the PR. But I was trying to understand the idea behind having full-width to the container at some places like offering 

![image](https://user-images.githubusercontent.com/54600590/197840627-65cb8e8f-94fa-43a6-807b-a0562da78de6.png)

And not using the full-width here. 

![image](https://user-images.githubusercontent.com/54600590/197840779-8e1ee448-a492-4d4f-a3a5-c7c5e36ad136.png)

@ann-aot , what was decided from your conversations from @hellolynn-tbtb

Reply:
yes, whenever there is a database we need to use `full width`, and also for some forms like offering